statistics.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python3
+
+
+def _get_messages_count_by_sender(chat):
+    messages_count_by_sender = dict()
+    for message in chat.messages:
+        sender = message.sender
+        if sender in messages_count_by_sender:
+            messages_count_by_sender[sender] += 1
+        else:
+            messages_count_by_sender[sender] = 1
```

Comment:
Кстати, вместо такого, мне кажется, лучше сделать
```suggestion
        if sender not in messages_count_by_sender:
            messages_count_by_sender[sender] = 0
        messages_count_by_sender[sender] += 1
```

Reply:
Но это не обязательно, просто, как по мне, более удобный и более правильный способ